src/components/layout/header.tsx

```diff
@@ -2,34 +2,66 @@
 
 import Image from "next/image";
 import Link from "next/link";
+import AuthHeader from "@/components/auth/AuthHeader";
 import logo from "@public/images/logos/xplorer_logo.svg";
-import { usePathname } from "next/navigation";
-import clsx from "clsx";
 import { menus } from "./helper";
-import { Button } from "../ui/button";
 
-export const Header = () => {
-  const pathname = usePathname();
+const Header = () => {
 
-  return (
-    <section className="container mx-auto px-4">
-      <header className="flex items-center justify-between py-7">
-        <Image src={logo} alt="logo" />
-        <nav className="flex gap-4 sm:gap-10">
-          {menus.map(({ name, href }) => (
-            <Link
-              key={name}
-              href={href}
-              className={clsx({
-                "border-b-2 border-black": pathname === href,
-              })}
```

Comment:
hmm just wondering why is the styling for border line under the active menu is removed? 🤔 I don't mind the removal of this style however would be better to show some kind of visibility of which page are we on atm. What do you think?

Reply:
It was by mistake i guess. Fixed